src/renderer/components/wallet/txs/receive/Receive.tsx

```diff
@@ -103,7 +86,7 @@ export const Receive: React.FC<Props> = (props): JSX.Element => {
           <Styled.Div>
             <label htmlFor="clipboard-btn">
               <Styled.Address size="large">
-                {isDesktopView ? addressLabel : truncateAddress(addressLabel, chain as Chain, network)}
+                {!isDesktopView && chain ? truncateAddress(addressLabel, chain, network) : addressLabel}
```

Comment:
Instead of 
```
chain ? truncateAddress(addressLabel, chain, network) : addressLabel
```

you can provide a `renderAddress` function as follow:
```
const renderAddress = useMemo(() => {
  FP.pipe(
    oChain,
    O.fold(
      () => addressLabel,
      (chain) => truncateAddress(addressLabel, chain, network)
  )

}, [oChain])
```